imports/api/rest/post-process-db-change-message.js

```diff
@@ -16,28 +18,82 @@ export default (req, res) => {
 
   const {
     notification_type: type,
-    invitee_user_id: inviteeId,
     case_title: caseTitle,
     case_id: caseId
   } = message
 
+  var emailAddr, emailContent, assigneeId, assignee
+
   switch (type) {
+    case 'case_new':
+      // https://github.com/unee-t/sns2email/issues/1
+      // When a new case is created, we need to inform the person who is assigned to that case.
+
+      assigneeId = message.assignee_user_id
+      assignee = Meteor.users.findOne({'bugzillaCreds.id': parseInt(assigneeId)})
+      if (!assignee) {
+        console.error('Could deliver message to missing user of BZ ID: ' + assigneeId)
+        return
+      }
+      emailAddr = assignee.emails[0].address
```

Comment:
You can and should eliminate a lot of the code duplication by taking these lines out of the switch cases